adapters/Pangle/PangleAdapter/GADMediationAdapterPangle.m

```diff
@@ -63,22 +106,57 @@ + (GADVersionNumber)adapterVersion {
 
 - (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration
                    completionHandler:(GADMediationBannerLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding banner ad.
+    
+    [GADMediationAdapterPangle setCOPPA:(adConfiguration.childDirectedTreatment ? adConfiguration.childDirectedTreatment.integerValue : -1)];
+    _bannerRenderer = [[GADPangleRTBBannerRenderer alloc] init];
+    [_bannerRenderer renderBannerForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
 - (void)loadInterstitialForAdConfiguration:(GADMediationInterstitialAdConfiguration *)adConfiguration
                          completionHandler:(GADMediationInterstitialLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding interstitial ad.
-}
-
-- (void)loadNativeAdForAdConfiguration:(GADMediationNativeAdConfiguration *)adConfiguration
-                     completionHandler:(GADMediationNativeLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding native ad.
+    [GADMediationAdapterPangle setCOPPA:(adConfiguration.childDirectedTreatment ? adConfiguration.childDirectedTreatment.integerValue : -1)];
+    _interstitialRenderer = [[GADPangleRTBInterstitialRenderer alloc] init];
+    [_interstitialRenderer renderInterstitialForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
 - (void)loadRewardedAdForAdConfiguration:(GADMediationRewardedAdConfiguration *)adConfiguration
                        completionHandler:(GADMediationRewardedLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding rewarded ad.
+    [GADMediationAdapterPangle setCOPPA:(adConfiguration.childDirectedTreatment ? adConfiguration.childDirectedTreatment.integerValue : -1)];
+    _rewardedRenderer = [[GADPangleRTBRewardedRenderer alloc] init];
+    [_rewardedRenderer renderRewardedAdForAdConfiguration:adConfiguration completionHandler:completionHandler];
+}
+
```

Comment:
Add these comments and I don't think we should expose this API to the publishers.
```
/// Set the COPPA setting in Pangle SDK.
///
/// @param COPPA An integer value that indicates whether the app should be treated as
/// child-directed for purposes of the COPPA.  0 means false. 1 means true. -1 means
/// unspecified. Any value outside of -1, 0, or 1 will result in this method being a no-op.
```